include/nbl/builtin/hlsl/sort/common.hlsl

```diff
@@ -19,6 +19,7 @@ struct CountingParameters
 
     uint32_t dataElementCount;
     uint32_t elementsPerWT;
+    uint32_t workGroupIndex;
     Key minimum;
     Key maximum;
```

Comment:
its a little weird to shove non-uniform (ones that change between workgroups) parameters together with  uniform

Reply:
Done